packages/mantle/src/mantle.css

```
@@ -588,6 +588,83 @@ MARK: UTILITIES
+		transform: translate3d(0, 0, 0) scale(1);
+		filter: blur(0);
+	}
+	to {
+		opacity: 0;
+		transform: scale(0.95);
+		filter: blur(4px);
+	}
+}
+
+@utility overlay-animation {
+	&[data-side="top"] {
```

Stylelint (17.12.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)